libs/domain/picking/offline/services/adapter/picking-list-online-default.adapter.spec.ts

```diff
@@ -127,7 +129,25 @@ describe('PickingListOnlineAdapter', () => {
     it('should parse picking lists', async () => {
       await nextTick(7);
 
-      expect(callback).toHaveBeenCalled();
+      expect(callback).toHaveBeenCalledWith([
+        expect.objectContaining({
+          items: [
+            {
+              id: 'mock',
+              orderItem: { uuid: 'mock' },
+              type: 'picking-list-items',
+              status: ItemsFilters.NotPicked,
+              productId: 'mock',
+              quantity: 1,
+            },
+          ],
+          itemsCount: 1,
+          requestedDeliveryDate: new Date(123),
+          orderReferences: ['mock'],
+          localStatus: 'mock',
+          productSkus: ['mock'],
```

Comment:
It's better to use more specific values so that the test is more precise and also the error messages are more clear when they will happen.
You can add specific text to each like `mock-order-id` or `mock-status`.